openstack/client.go

```diff
@@ -31,17 +32,21 @@ func NewClient(endpoint string) (*gophercloud.ProviderClient, error) {
 	endpoint = gophercloud.NormalizeURL(endpoint)
 	base = gophercloud.NormalizeURL(base)
 
+	pc := &gophercloud.ProviderClient{
+		IdentityBase: base,
+		Logger: &logrus.Logger{
```

Comment:
Should we (as above) initialize a logger for users who create a client or let them create it themselves?